packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -120,8 +120,10 @@ def create_pool(
         worker_pool = WorkerPool(
             name=name,
             max_count=number,
-            image=worker_image,
+            image_id=worker_image.id,
             worker_list=worker_list,
+            syft_node_location=context.node.id,
```

Comment:
added this because otherwise 
```python
api = APIRegistry.api_for(
            node_uid=self.syft_node_location,
            user_verify_key=self.syft_client_verify_key,
        )
```
would output `None`